mkShapesRDF/shapeAnalysis/latinos/LatinosUtils.py

```diff
@@ -8,12 +8,14 @@ def flatten_samples(samples):
 
         subsamplesmap.append((sname, []))
         for sub in sample["subsamples"]:
-            samples["%s_%s" % (sname, sub)] = sample
-            subsamplesmap[-1][1].append("%s_%s" % (sname, sub))
+            samples["%s" % (sub)] = sample
```

Comment:
Not sure what the problem is, this is coming from original LatinoAnalisys code [here](https://github.com/latinos/LatinoAnalysis/blob/c2d28b18160be88cced2bc38ae5a4310c04fc526/ShapeAnalysis/python/utils.py#L9-L15)

Reply:
The code will always prepend the "sample" name to the "subsample" name. Suppose you have a sample, and you split into subsamples with different event weight then the subsample name is not user defined and that could affect other part of the process, like tayloring the datacrds to combine models (e.g you have a sample "Zee" and you want it into the "sm" subsample, "quad_cW" subsample, .... you'll never be able to have the correct naming).
By lifting this the user has full flexibility on the subsamples names and can alsp prepend the sample name if he wants to

Reply:
Wouldn't the final names be `['Zee_sm', 'Zee_quad_cW']` ?